server/graphql/common/orders.js

```diff
@@ -75,6 +75,10 @@ export async function addFunds(order, remoteUser) {
     orderData.data.memo = order.memo;
   }
 
+  if (!isNil(order.fundReceivedDate)) {
+    orderData.data.fundReceivedDate = order.fundReceivedDate;
+  }
+
```

Comment:
Not a fan of putting this one in `data`. It seems like something we're going to set for all `Orders` and that we may want to filter/order on in the future. So a new column makes sense.

Could we re-use `order.processedAt` for that? I think the current definition of the column matches what we want.

Reply:
Makes sense. I've re-used `order.processedAt`. There was a few changes that needed to be done in the [`payments.js`](https://github.com/opencollective/opencollective-api/blob/759432476ab64df434596b25dcc7a768e2d50090/server/lib/payments.js#L548) as otherwise the `processedAt` date will be overridden.